.husky/post-checkout

```diff
@@ -0,0 +1,4 @@
+#!/usr/bin/env sh
+. "$(dirname -- "$0")/_/husky.sh"
```

Comment:
I think this line deserves a comment.

Reply:
It was generated from husky, so I haven't really given any thought into what it actually does.